app/experimenter/experiments/constants.py

```diff
@@ -194,7 +194,7 @@ class ExperimentConstants(object):
     FX_MIN_MULTI_BRANCHED_VERSION = 68
 
     # ExperimentVariantName REGEX
-    EXPERIMENT_VARIANT_NAME_REGEX = re.compile(r"^[a-z0-9 ]+$", re.IGNORECASE)
+    EXPERIMENT_VARIANT_NAME_REGEX = re.compile(r"^[a-z0-9 \-]+$", re.IGNORECASE)
```

Comment:
Wait do we still need this?

Reply:
no, planning to remove it in https://github.com/mozilla/experimenter/pull/2051 after this was merged